project_depatment/application/static/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
static is not a package - init.py is redundant

Reply:
fixed